R/pkg/R/generics.R

```diff
@@ -918,6 +918,10 @@ setGeneric("explode_outer", function(x) { standardGeneric("explode_outer") })
 #' @name NULL
 setGeneric("expr", function(x) { standardGeneric("expr") })
 
+#' @rdname column_collection_functions
+#' @name NULL
+setGeneric("flatten", function(x, value) { standardGeneric("flatten") })
```

Comment:
`function(x) { standardGeneric("flatten") }`?

Reply:
Thanks for catching the problem. Will correct.

Reply:
great catch!